### DISCUSSION TITLE
NavMesh 사용하시는 분 있나요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#483] [5. Q&A]
NavMesh에서 SamplePosition을 사용하려고 하면 계속 에러가 뜨는 데, 
혹시 성공하신 분 참고하신 예제 같은 게 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 어떤 에러가 뜨는지 
구현한 코드와 맵의 구조, NavMesh를 적용한 스크린샷과 에러 스크린샷 등을 첨부해주시면 도움을 드릴 수 있을 것 같습니다.

**Comment by ryusiin** [comment]
해결했습니다:>
![image](https://user-images.githubusercontent.com/5895396/158944661-cb3831e8-6dc9-4115-acd7-f712ee9ddfe2.png)
